### DISCUSSION TITLE
Cant apply to education programm

### DISCUSSION THREAD

**Discussion by Lucatschy** [community/community#46051] [GitHub Education]
I live in Germany and my school (gymnasium königsbrunn) doesnt give u an .edu email but an @gyk.bayern email(bayern is the "state" i live in).But i also doesnt have and becauseg of that i wanted to send a picture of out school portal(teams).But then the website says:
![Request a discount - GitHub Education – Mozilla Firefox 01 02 2023 21_03_52](https://user-images.githubusercontent.com/96542090/216151129-6e40bdb3-fb19-4170-b7d4-55277544aa03.png)
my screenshot of teams where you can clearly see the school name:
![Bildschirmfoto 2023-02-01 um 20 37 54](https://user-images.githubusercontent.com/96542090/216151223-74fc83bb-7c01-48da-98cd-91f961f7abd4.jpg)
You can clearly know that im a student because otherwise i wont be at the school platform

**Comment by SilasRodrigues19** [comment]
Hey @Lucatschy 

You can request assistance through this link here:

[GitHub Education Support](https://support.github.com/request/education?tags=hubberfy_education)

**Comment by jge162** [comment]
> I live in Germany and my school (gymnasium königsbrunn) doesnt give u an .edu email but an @gyk.bayern email(bayern is the "state" i live in).But i also doesnt have and becauseg of that i wanted to send a picture of out school portal(teams).But then the website says:
> ![Request a discount - GitHub Education – Mozilla Firefox 01 02 2023 21_03_52](https://user-images.githubusercontent.com/96542090/216151129-6e40bdb3-fb19-4170-b7d4-55277544aa03.png)
> my screenshot of teams where you can clearly see the school name:
> ![Bildschirmfoto 2023-02-01 um 20 37 54](https://user-images.githubusercontent.com/96542090/216151223-74fc83bb-7c01-48da-98cd-91f961f7abd4.jpg)
> You can clearly know that im a student because otherwise i wont be at the school platform

Upload an image of your transcripts or school Id. Teams with email logged in probably not a good option.
